test/semantics/separate_module_procs_1.f90

```diff
@@ -0,0 +1,61 @@
+! Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Change this to your company's (AMD?) copyright.

Reply:
Almost all files had NVIDIA copyright, so i did not try to change it.
I will go ahead and use the new license for the files that i added.

Reply:
For what it's worth, we've been typically using hyphens (`-`) rather than underscores (`_`) in file names.